ui/js/graphs.js

```diff
@@ -684,20 +684,21 @@ perf.controller('GraphsCtrl', [
             });
 
             modalInstance.result.then(function(series) {
-              series.highlightedPoints = [];
-              series.visible = true;
-              series.color = availableColors.pop();
-
-              $scope.seriesList.push(series);
-              if( !$scope.highlightedRevision ) {
-                $scope.highlightedRevision = '';
-              }
-              if (!$scope.zoom) {
-                $scope.zoom = {};
+              for (var i = 0; i < series.length; i++) {
```

Comment:
I'd use lodash's foreach here, instead of a for loop:

https://lodash.com/docs#forEach